src/styles/blocks/main/for-whom.scss

```
@@ -0,0 +1,275 @@
+.for-whom {
+  padding: 0 $sidePaddingMobile;
+
+  &__title-2 {
+    margin-bottom: $marginInside;
+  }
+}
+
+.lot-1,
+.lot-2,
+.lot-3 {
+
+  &__photo {
+    display: none;
+    margin-right: 10px;
```

```suggestion
    margin-right: 10px;
```
Is it a block or modifier? If it's a block, so it cannot have outer spacing. If modifier, so it should be named with `--` or `_` (`lot--1`)